keras/objectives.py

```diff
@@ -11,6 +11,10 @@ def mean_absolute_error(y_true, y_pred):
     return K.mean(K.abs(y_pred - y_true), axis=-1)
 
 
+def mean_identity_error(y_true, y_pred):
+    return K.mean(y_pred, axis=-1)
```

Comment:
It's a hack, but changing to something like 

```
return K.mean(y_pred, axis=-1) + (0 * y_true)
```

may suppress the Theano warning that you're currently throwing.